functions-and-scope.js

```diff
@@ -15,38 +23,119 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6
-
+console.log("opdracht 1a, hoeveel studenten zijn Cum Laude geslaagd?")
+let studentCumLaude = 0;
+for (let participants = 0; participants < grades.length; participants++) {
+    if (grades[participants] > 7) {
+        studentCumLaude = studentCumLaude + 1
+    }
+}
+console.log("We hebben "+ studentCumLaude + " speciale cum laude diploma's nodig.");
 
 /*  1b: Omschrijven tot een herbruikbare functie   */
 // Schrijf een functie genaamd cumLaude, die een array van cijfers verwacht (zoals grades) en het aantal Cum laude studenten teruggeeft. Gebruik hiervoor jouw antwoord van 1a.
 // Zorg ervoor dat jouw functie ook werkt als we een andere array met eindcijfers willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+console.log("opdracht 1b maar nu als een herbruikbare functie:")
+
+function cumLaude (gradeArray) {
+    let studentCumLaude = 0;
+    for (let participants = 0; participants < gradeArray.length; participants++) {
+        if (gradeArray[participants] > 7) {
+            studentCumLaude = studentCumLaude + 1
+        }
+    }
+    return studentCumLaude;
+}
+cumLaude(grades);
+
+console.log(studentCumLaude);
+
+//wat als ik meerdere verzamelingen van grades heb? Bijvoorbeeld: grades1969 en grades1970?
+//hieronder een voorbeeld hoe je de functie voor meerdere arrays (grades en grades2) kan toepassen:
+
+console.log("opdracht 1c functie geschikt voor meerdere arrays met elk hun eigen naaam")
+
+//grades is al eerder gedefinieerd
+studentCumLaude = 0; //let is niet meer nodig want al eerder gedeclareerd
```

Comment:
`studentCumLaude` is gedeclareerd binnen een functie, dus die zou hier wél een `let` ervoor moeten krijgen. Echter ben ik van mening dat hij hier óók binnen de functie moet komen. En dan is de functie `cumLade` exact hetzelfde als `cumLaude`.